storage/EtcCrypttab.cc

```diff
@@ -157,8 +156,10 @@ namespace storage
     }
 
 
-    EtcCrypttab::EtcCrypttab(const string& filename) :
-	ColumnConfigFile()
+    const int EtcCrypttab::DEFAULT_PERMISSIONS = 0640;
+
+    EtcCrypttab::EtcCrypttab(const string& filename, const int permissions) :
```

Comment:
`const` for function parameters only makes sense if it's a const reference, not for basic types like _int_. Please remove the `const` here.

Reply:
It is not only overkill, it is also against all C++ conventions. :smile: 

Any modification to the parameter will not be visible to the caller anyway because it's call by value.